recipe/meta.yaml

```diff
@@ -22,13 +22,14 @@ build:
 
 requirements:
   host:
-    - python >=3.5
+    - python >=3.6
     - pip
     - nodejs >=10
   run:
-    - python >=3.5
-    - jupyterlab_server >=1.0.0,<2.0.0
-    - notebook >=4.3.1
+    - python >=3.6
+    - jupyterlab_server >=2.0.0rc1,<3.0.0
+    - jupyter_server >=1.0.1,<2.0.0
+    - nbclassic >=0.2.0,<1.0.0
```

Comment:
according to semver, 0.3 could have breaking changes from 0.2. Not sure what we are following with nbclassic, though.

(well, technically any pre-1.0 release can do whatever it wants, but the typical ^ implies breaking changes possible between 0.2 and 0.3

All this to say that either nbclassic should probably either bump to 1.0, or we should restrict to 0.2.0 to `<0.3.0`.

Reply:
Yes `nbclassic >=0.2.0,<0.3.0` sounds good for now. Unless it's ready for 1.0.